### DISCUSSION TITLE
Support Firefox

### DISCUSSION THREAD

**Discussion by johannesCmayer** [coddingtonbear/obsidian-web#129] [Ideas]
Hi,

I find this extension very useful and use it all the time. Because adblockers don't work for <chrome/brave> anymore I am considering switching to Firefox. I don't know how hard it would be to make this extension work with Firefox, though that would be nice. All the other extensions for obsidian clipping suck. This does exactly what I want.

**Comment by coddingtonbear** [comment]
This is a duplicate of #79
